meilisearch-http/src/task.rs

```
@@ -79,15 +84,19 @@ pub enum TaskStatus {
 }
 
 impl FromStr for TaskStatus {
-    type Err = &'static str;
+    type Err = String;
 
-    fn from_str(status: &str) -> Result<Self, &'static str> {
+    fn from_str(status: &str) -> Result<Self, String> {
```

Same here, I think we need to return a custom error.